tests/test_url_query.py

```diff
@@ -171,3 +171,22 @@ def test_query_from_empty_update_query(
 
     if "b" in original_url.query:
         assert new_url.query["b"] == original_url.query["b"]
+
+
+@pytest.mark.parametrize(
+    "original_query_string, keys_to_drop, expected_query_string",
```

Comment:
nit: it's generally more readable to use the following param declaration:
```suggestion
    ("original_query_string", "keys_to_drop", "expected_query_string"),
```